From ce5dbab99bd9c789f5729e1de8a5963eb7eec598 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Mon, 9 Dec 2019 14:57:44 -0700
Subject: [PATCH 06/22] xen/blkfront: Remove spaces before labels

The general C convention is that labels do not have any indentation.

Suggested-by: Nick Desaulniers <ndesaulniers@google.com>
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/block/xen-blkfront.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/block/xen-blkfront.c b/drivers/block/xen-blkfront.c
index 57d50c5ba309..3441b281a6c8 100644
--- a/drivers/block/xen-blkfront.c
+++ b/drivers/block/xen-blkfront.c
@@ -1175,9 +1175,9 @@ static int xlvbd_alloc_gendisk(blkif_sector_t capacity,
 
 	return 0;
 
- release:
+release:
 	xlbd_release_minors(minor, nr_minors);
- out:
+out:
 	return err;
 }
 
@@ -1557,7 +1557,7 @@ static irqreturn_t blkif_interrupt(int irq, void *dev_id)
 		return IRQ_HANDLED;
 
 	spin_lock_irqsave(&rinfo->ring_lock, flags);
- again:
+again:
 	rp = rinfo->ring.sring->rsp_prod;
 	rmb(); /* Ensure we see queued responses up to 'rp'. */
 
@@ -1883,11 +1883,11 @@ static int talk_to_blkback(struct xenbus_device *dev,
 
 	return 0;
 
- abort_transaction:
+abort_transaction:
 	xenbus_transaction_end(xbt, 1);
 	if (message)
 		xenbus_dev_fatal(dev, err, "%s", message);
- destroy_blkring:
+destroy_blkring:
 	blkif_free(info, 0);
 
 	mutex_lock(&blkfront_mutex);
-- 
2.24.1

